src/client/components/editCropImage/editCrop.component.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import './editCrop.style.css';
+import PropTypes from 'prop-types';
+import penImage from '../../assets/images/edit-icon.png';
+
+export default function Image({ srcPath, altText }) {
+  return (
+    <div>
+      <img className="edit-image" src={srcPath} alt={altText} />
+      <button type="submit" className="edit-icon">
+        <img src={penImage} alt="editImage" />
```

Comment:
If you set `background: none;` for this icon, it will match the design (its background is gray now). 👍

Reply:
I changed the background and it worked thank you